src/main.module.ts

```diff
@@ -42,10 +42,16 @@ import { LoggingInterceptor } from './logging/logging-interceptor.helper';
             clientRequest: {
               logKey: 'req',
               headersLogKey: 'headers',
+              bodyLogKey: 'data',
             },
             outgoingRequest: {
               logKey: OUTGOING_REQUEST_LOG_KEY,
               headersLogKey: HEADERS_LOG_KEY,
+              bodyLogKey: BODY_LOG_KEY,
+            },
+            incomingResponse: {
+              logKey: INCOMING_RESPONSE_LOG_KEY,
+              bodyLogKey: BODY_LOG_KEY,
```

Comment:
Is this removing full incoming response body? or just one key in the JSON?
Body still might be nice to log for debugging.

Is this removing full outgoing response body? or just one key in the JSON?
Because Post/body is new functionality for http client it might be fine to remove full body from the log.

Reply:
Good point - I've specified just the access token rather than the whole body to redact in the incoming response, and removed the outgoing request change as it's actually not necessary anymore, was from a previous implementation.